dataflux_pytorch/benchmark/README.md

```diff
@@ -25,6 +25,8 @@ Then set the command line variables.
 
 `--layers`: you can also optionally change the size of the model. The `--layers` argument will be passed into `nn.Transformer` for `num_encoder_layers` and `num_decoder_layers`. The default value for `--layers` is 100.
 
+`--clear-kernel-cache`: this is an optional argument which when set clears kernel cache after saving the checkpoints. If benchmarking gcsfuse, this argument must be set.
```

Comment:
I'd recommend making the "If benchmarking" language a bit more generic. While we ran into this when trying out gcsfuse, it would likely apply to any filesystem, for example when saving to local disk. In fact now that I think about it that's probably why my Local SSD tests I ran before had incredibly fast read times

Reply:
Changed it to "If benchmarking local filesystems or storage solutions that expose a filesystem interface (such as gcsfuse), this argument must be set in order to get more accurate performance measurements."

Does that look good?

Reply:
Looks good. Something else I realized is that will only work on Linux/Mac as-is, and may only work when running with sudo. Might need to document that also.

Reply:
Done! Made it so that the command to clear cache is run only if the OS is Linux or MacOS. Also added a note in the README about that. Does it look good?

Reply:
Great, thanks